src/System.Text.Utf8/tests/RandomTests.cs

```diff
@@ -320,5 +321,168 @@ public void SubstringToCodeUnit(string s, char c)
                 Assert.Equal(expected, u8result.ToString());
             }
         }
+
+        [Theory]
+        [InlineData("")]
+        [InlineData("a")]
+        [InlineData("ab")]
+        [InlineData("abcdefghijklmnopqrstuvwxyz")]
+        [InlineData("ABCDEFGHIJKLMNOPQRSTUVWXYZ")]
+        [InlineData("0123456789")]
+        [InlineData(" ,.\r\n[]<>()")]
+        [InlineData("1258")]
+        [InlineData("1258Hello")]
+        [InlineData("\uABCD")]
+        [InlineData("\uABEE")]
+        [InlineData("a\uABEE")]
+        [InlineData("a\uABEEa")]
+        [InlineData("a\uABEE\uABCDa")]
+        public void CodePointEnumeratorsTests(string s)
```

Comment:
I'm currently at the point where I want to quickly test and modify features, I find this the fastest and most reliable. Whenever I change the code (not talking about simple rename) I need to change a single place instead of multiple tests. I'm planning to write better tests when we will be hardening the implementation or will be at least on more stable design stage.